src/arrayMethodSort.js

```diff
@@ -4,8 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    return a.toString() > b.toString();
+  }) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) => a.toString() > b.toString() ) {
```

Reply:
I cant do like that bcs max length in linter === 80

Reply:
Then move to a new line
```suggestion
  [].__proto__.sort2 = function(
  compareFunction = (a, b) =>  a.toString() > b.toString();
  ) {
```